pom.xml

```diff
@@ -164,6 +164,25 @@
                             <finalName>InteropServiceExampleApplication</finalName>
                         </configuration>
                     </execution>
+                    <execution>
+                        <id>JavaFDC3Example</id>
+                        <phase>package</phase>
+                        <goals>
+                            <goal>shade</goal>
+                        </goals>
+                        <configuration>
+                            <createDependencyReducedPom>false</createDependencyReducedPom>
+                            <transformers>
+                                <transformer
+                                        implementation="org.apache.maven.plugins.shade.resource.ManifestResourceTransformer">
+                                    <manifestEntries>
+                                        <Main-Class>com.finsemble.example.fdc3.JavaFDC3Example</Main-Class>
```

Comment:
Should this be com.finsemble or io.interop?

Reply:
That makes more sense, I think. I'll make that change.